.github/workflows/test-s390x.yml

```diff
@@ -0,0 +1,92 @@
+name: test-s390x
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
+  pull_request:
+    branches:
+      - main
+      - 'release/**'
+
+jobs:
+  test-linux-s390x:
+    if: (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/release*')
+    runs-on: ubuntu-latest
+    env:
+      ZVSI_FP_NAME: bp-floating-ci-${{ github.run_id }} 
+      ZVSI_INSTANCE_NAME: bp-zvsi-ci-${{ github.run_id }} 
+      ZVSI_VPC: r038-b86410fb-6889-4094-821d-1ceb72176bc4
+      ZVSI_ZONE_NAME: ca-tor-1
+      ZVSI_PROFILE_NAME: bz2-4x16
+      ZVSI_SUBNET: 02q7-af49cf76-0471-49f4-a245-b60d63196efb
+      ZVSI_IMAGE: r038-67da7290-933e-42e4-a1c2-ff439b4b6d2c
+      ZVSI_KEY: r038-f8e35cee-0475-4315-a051-40f4d2fb6af9
+      ZVSI_RG_ID: 31e0f1b03d1b4d42bd98b6768ba5421a
+      ZVSI_SG: r038-1e644f46-8044-4043-9108-2b7dc178827f
```

Comment:
Is there an argument for putting any of these in secrets instead of hard coding?